src/coreimage.cs

```diff
@@ -2768,22 +2916,28 @@ interface CICodeGenerator {
 	[iOS (8,0)]
 	[Mac (10,10)]
 	[BaseType (typeof (CICodeGenerator))]
-	interface CIAztecCodeGenerator {
+	interface CIAztecCodeGenerator : ICIAztecCodeGeneratorProtocol {
 
 		[CoreImageFilterProperty ("inputCompactStyle")]
 		bool CompactStyle { get; set; }
 
+		[CoreImageFilterProperty ("inputCompactStyle")]
+		float FCompactStyle { get; set; }
```

Comment:
What about `InputCompactStyle`?

If not, then I think I prefer `FloatCompactStyle`, it's just as ugly as `FCompactStyle`, but more informative (and the same goes for all the other new float properties).

Reply:
yeah, ideally there's a common naming fix that can be used - at least for the `F` and `N` prefixes. I'll check if `Input` works well on all of them :)